README.md

```diff
@@ -1,3 +1,5 @@
 # hackpack-bot
 NC State's Hackpack Discord Bot, written in Go.
-Maintained by John Allison (john123allison@gmail.com)
+Created by John Allison (john123allison@gmail.com)
+
+Maintained by Michael Blunt (mbblunt@ncsu.edu)
```

Comment:
also, while it is accurate that I will/am handing over maintenance to the active HackPack team, it's generally good practice to ask directly before making a change like this. No harm no foul, just a heads up.

Reply:
Sorry! Didn't mean anything bad by that, thanks for letting me know. This is my first PR, still a little rough on the edges with the etiquette :)

Reply:
All good, don't sweat it ! :)